tests/test_abortable.py

```diff
@@ -38,19 +39,33 @@ def do_work() -> None:
     # If I send it a message
     message = do_work.send()
 
-    # Then wait and signal the task to terminate
-    time.sleep(0.1)
-    abort(message.message_id)
+    if mode == 'abort':
```

Comment:
Make separate test and drop the `mode` fixture.

There's no point using parameters fixtures if we end up doing a bunch of `if` inside the test.